how_to_build.md

```diff
@@ -94,6 +94,7 @@ available:
 * `net45`: builds only the .NET Framework 4.5 target framework
 * `netcore2.1`: builds only .NET Core 2.1 / .NET Standard 2.0 target frameworks
 * `netcore3.1`: builds only .NET Core 3.1 / .NET Standard 2.1 target frameworks
+* `net5.0`: builds only .NET 5.0 target frameworks
 * `netcoreall`: the default profile on non-Windows OSes, builds all supported
   .NET Core / .NET Standard target frameworks

```

Comment:
Is 5.0 technically .NET Core? I don't mind 5.0 building from the `netcoreall` profile, but wonder if it wouldn't be better to mention 5.0 explicitly here and elsewhere where we talk about the supported TFMs.

Also, up on lines 84/85/86, we don't say that we build on .NET 5.0 (nor do we say we test on it, unless you think .NET Core _does_ include .NET 5.0…)

Reply:
> Is 5.0 technically .NET Core?

Good question. "Technically", I'd say yes. The removal of "Core" from the name is just branding; the technology remains fundamentally the same as .NET Core. But it's true that the name `netcoreall` doesn't make it obvious that it includes .NET 5.0.

So... `netcoreand5.0` ?
Or maybe `fullexceptnetfx` ? i.e. instead of mentioning all included TFMs, we just mention the one that is excluded

Another possible approach: change `full` so that it includes net45 on Windows but not on Linux/MacOS. This way, the default profile is always `full`. And, as an added benefit, we don't need to come up with an awkward name for `netcoreall`.

> Also, up on lines 84/85/86, we don't say that we build on .NET 5.0 (nor do we say we test on it, unless you think .NET Core does include .NET 5.0…)

That's an oversight, I'll fix it.

Reply:
> Another possible approach: change full so that it includes net45 on Windows but not on Linux/MacOS

I like it.

Reply:
> I like it.

Great. I'll see if it's possible (I don't see why not, but who knows...)